frontend/public/components/utils/index.tsx

```diff
@@ -78,3 +80,19 @@ export const enum EnvType {
   ENV = 0,
   ENV_FROM = 1
 }
+
+export type FirehoseResource = {
```

Comment:
We should update `details.tsx` to use this:

https://github.com/openshift/console/blob/ec544a37552229b836a4970c9c5d222a82d240ff/frontend/public/components/factory/details.tsx#L10

Reply:
That is already updated - see https://github.com/openshift/console/pull/1696/files#diff-c09a5bef454710fc5ba83321c2e0fe1fR5